servers/partner-ci/9.0/manila.yaml

```diff
@@ -176,6 +204,7 @@
           kvm_use:              '{kvm_use}'
           iso_storage:          '{iso_storage}'
           snapshots_id:         '{snapshots_id}'
+      - plugin-specific-parameters
```

Comment:
to initialize **plugin-specific-parameters** need to pass all variables defined above in the **project** section